docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,70 @@
 # Parameters
 
-!!! warning
+Autoware has following parameters
 
-    Under Construction
+- `p1`: reference parameter for the node package
+  - e.g., [the parameter for the `behavior_path_planner` package](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/behavior_path_planner/config)
+- `p2`: reference parameter for the module launch
```

Comment:
I referred tier4_*_launch to the module launch. I'm wondering about a better name.